tools/git_reset_generated_dirs.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+# HOW TO USE:
```

Comment:
You could use a `Note [git generated dirs]` script to avoid having to copy paste it two places ;)